dev-docs/new-distro.md

```diff
@@ -39,10 +39,12 @@ manifest as runtime errors that won't show up until tests are run.
 
 1.  Temporarily repurpose one of the existing Kokoro builds for testing
     your new distro. For example, in
-    [build/presubmit/bullseye.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/build/presubmit/bullseye.gcl),
+    [kokoro/config/build/presubmit/bullseye_x86_64.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/build/presubmit/bullseye_x86_64.gcl)
+    or [kokoro/config/build/presubmit/bullseye_aarch64.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/build/presubmit/bullseye_aarch64.gcl),
     replace `bullseye` with `$DISTRO_SHORT` and change `deb` to `rpm` if
     needed. Then in
-    [test/ops_agent/bullseye.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/test/ops_agent/bullseye.gcl),
+    [kokoro/config/test/ops_agent/bullseye_x86_64.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/test/ops_agent/bullseye_x86_64.gcl)
+    or [kokoro/config/test/ops_agent/bullseye_aarch64.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/test/ops_agent/bullseye_aarch64.gcl),
```

Comment:
Maybe also update line 61 to say: `bullseye_$arch.gcl`

Reply:
Done.